### DISCUSSION TITLE
Issue in running app in android

### DISCUSSION THREAD

**Discussion by rachna-khatnawlia** [react-native-community/discussions-and-proposals#682] [Ideas]
Creating react native project with latest version.

In IOS its running fine, but while running in android i am getting the following errror:

<img width="1440" alt="Screenshot 2023-07-05 at 12 58 46 PM" src="https://github.com/react-native-community/discussions-and-proposals/assets/60135781/7f1f7e4e-72bf-4a35-b1e3-7ed205adcf09">

result of npx react-native info:
System:
  OS: macOS 13.2.1
  CPU: (8) arm64 Apple M2
  Memory: 125.19 MB / 8.00 GB
  Shell:
    version: 5.8.1
    path: /bin/zsh
Binaries:
  Node:
    version: 19.7.0
    path: /opt/homebrew/bin/node
  Yarn: Not Found
  npm:
    version: 9.5.0
    path: /opt/homebrew/bin/npm
  Watchman:
    version: 2023.02.27.00
    path: /opt/homebrew/bin/watchman
Managers:
  CocoaPods:
    version: 1.12.1
    path: /Users/rachna/.rvm/gems/ruby-2.7.6/bin/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 22.2
      - iOS 16.2
      - macOS 13.1
      - tvOS 16.1
      - watchOS 9.1
  Android SDK: Not Found
IDEs:
  Android Studio: 2022.1 AI-221.6008.13.2211.9514443
  Xcode:
    version: 14.2/14C18
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 11.0.18
    path: /usr/bin/javac
  Ruby:
    version: 2.7.6
    path: /Users/rachna/.rvm/rubies/ruby-2.7.6/bin/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.1
    wanted: 0.72.1
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: true
  newArchEnabled: false

**Comment by kelset** [comment]
👋 there - as stated in the README and the issue template, this repo is for long-form discussions. You should be posting this issue on the main [react native repo](https://github.com/facebook/react-native).
